geth/common/utils.go

```diff
@@ -65,26 +72,52 @@ func ToAddress(accountAddress string) *common.Address {
 	return &to.Address
 }
 
-// ImportTestAccount checks if test account exists in keystore, and if not
-// tries to import it (from static resources, see "static/keys" folder)
-func ImportTestAccount(keystoreDir, accountFile string) error {
-	// make sure that keystore folder exists
-	if _, err := os.Stat(keystoreDir); os.IsNotExist(err) {
-		os.MkdirAll(keystoreDir, os.ModePerm) // nolint: errcheck
+// RestoreFile checks if decrypted file exists in dir, and if not
+// tries to restore and decrypt it (from static resources, see "static/keys" folder)
+func RestoreFile(dir, file string) error {
```

Comment:
Maybe it would be better to move this function and it's siblings to a separate package (can be a TODO), let's say bindata, and rename it to Extract:
bindata.Extract(name, path string) error